src/components/ClusterOverview/ClusterOverview.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Card, CardTitle, CardBody, CardHeading, Popover, Button, Icon, OverlayTrigger, Col } from 'patternfly-react';
+
+import { Details } from './Details/Details';
+import { Health, Compliance } from './Health/Health';
+import { Events } from './Events/Events';
+import { Inventory } from './Inventory/Inventory';
+import { Capacity } from './Capacity/Capacity';
+import { Utilization } from './Utilization/Utilization';
+import { Consumers } from './Consumers/Consumers';
+import { Loading } from '../Loading';
+
+const CardItemHelp = ({ content }) => {
+  const overlay = <Popover id="popover">{content}</Popover>;
+  return (
+    <OverlayTrigger overlay={overlay} placement="top" trigger={['click']} rootClose>
+      <Button bsStyle="link">
+        <Icon type="fa" name="info-circle" size="2x" />
+      </Button>
+    </OverlayTrigger>
+  );
+};
+
+CardItemHelp.propTypes = {
+  content: PropTypes.object.isRequired,
+};
+
+export const overviewItem = (LoadingComponent, Component, name) => props => (
+  <Card className={classNames('kubevirt-cluster-overview__item', `kubevirt-${name}`)}>
+    <CardHeading className="kubevirt-cluster-overview__item-heading">
+      <CardTitle>{Component.title}</CardTitle>
+      {Component.help && <CardItemHelp content={Component.help} />}
+    </CardHeading>
+    <CardBody className={`kubevirt-${name}__body`}>
+      {props.loaded ? <Component {...props} /> : <LoadingComponent />}
+    </CardBody>
+  </Card>
+);
+
+export class ClusterOverview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.DetailsItem = overviewItem(props.LoadingComponent, Details, 'details');
```

Comment:
As we discussed offline, this will be moved under new component plus wrapped into a Provider to supply data.